CHANGELOG.md

```diff
@@ -1,5 +1,76 @@
 # Change Log
 
+## [6.2.1.0]
+
+### Enhancements
+
+#### Build
+- **Common Library**: Arnold-usd now includes a set of common functions and string definitions to share across multiple modules. (#466)
+
+#### Procedural
+- **Half and Double precision**: Storing data using half or double precision is now supported. (#672)
+- **Velocity blur**: The procedural now uses the velocity attribute to create motion keys for point-based shapes, when there are no position keys or the topology changes between frames. (#221)
+- **NodeGraph schemas**: The procedural now supports using the NodeGraph schema for shader networks. (#678)
+- **Crease Sets**: The procedural now supports crease sets on polymesh. (#694)
+- **Purpose**: Usd Purpose is now supported in the procedural. (#698)
+- **Transform2D**: The procedural now supports remapping UsdTransform2D to built-in Arnold nodes. (#517)
+- **Multi-Threading**: The procedural now uses USD's WorkDispatcher which improves the performance of multi-threaded expansion in many cases. (#690)
+
+#### Render Delegate
+- **Half and Double precision**: Storing data using half or double precision is now supported. (#669)
+- **Light and Shadow linking**: The render delegate now supports light and shadow linking. (#412)
+- **Motion blur for the Point Instancer**: The render delegate now calculates motion blur when using the point instancer. (#653)
+- **Half and Double precision**: Storing data using half or double precision is now supported. (#669)
+- **Pause and Resume**: Pausing and resuming renders are now supported in the render delegate. (#595)
+
+#### Scene Format
+- **Write with default values**: The scene format now supports optionally writing parameters with default values. (#720)
```

Comment:
The `scope` and `all_atributes` keywords introduced in #721 should be documented somewhere, possibly as Doxygen comments, and mentioned in the release notes. CC @sebastienblor.

Reply:
Created #789.